### DISCUSSION TITLE
How to format a second custom menu like the Main Astroid Menu?

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#338] [Q&A]
If somebody want a second custom menu _(not the Main Menu choose in the Astroid manager > Header > Site Menu choice)_

Do you have a way to bring easily the same formatting _(top level items, chevron, sub-menus item, etc.)_

**If not, what will be the best way to achieve that?**

**Comment by Chacapamac** [comment]
@sonvnn 
You can see that people asking for the possibility to have a second menu formatted like the “Main Menu”. 
—> https://github.com/templaza/astroid-framework/discussions/336

It will be great if it was an easy way in Astroid to have a second menu formatted the exact same way as the “Main” menu without having to use coding or anything to complex.

**IDEAS:** 
- Set in Astroid Template Manager?
- Just by adding a class?
- having a special Astroid Menu module?

Ideally it should have a setting to merge or not to the Mobile Menu (With the Main Menu) with a way to place it before or after or in a certain order.

**Reply by sonvnn** [reply]
Hi,

I think the best way is create and Astroid Menu Module. I am working on it. I will inform you when I complete it.

Thanks for your idea.

Best Regards,
Sonny

**Reply by Chacapamac** [reply]
Follow Up on —> https://github.com/templaza/astroid-framework/discussions/336#discussioncomment-6072391
